content/applications/finance/fiscal_localizations/singapore.rst

```diff
@@ -0,0 +1,73 @@
+=========
+Singapore
+=========
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the :guilabel:`Singapore - Accounting` module to get the features
+of the SG localization.
+
+.. image:: singapore/sg-module.png
+   :align: center
+   :alt: "Singapore - Accounting" localization module marked as "installed"
+
+Add PayNow QR codes to invoices
+===============================
+
+PayNow is a payment service platform that allows customers to make instant domestic payments to
+individuals and merchants in Singapore dollars via online and mobile banking.
+
+Activate QR codes
+-----------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`. Under the :guilabel:`Customer
+Payments` section, activate the :guilabel:`QR Codes` feature.
+
+.. image:: singapore/sg-qr-code-setting.png
+   :align: center
+   :alt: "QR Codes" feature activated
+
+PayNow bank account configuration
+---------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
+which you want to activate PayNow. Set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy
+Value` field depending on the type you chose.
+
+.. note::
```

Comment:
important note, maybe? If not, you can group the next note with this one, in a bullet list
```suggestion
.. important::
```